### DISCUSSION TITLE
Oct. 29th Team Check-In

### DISCUSSION THREAD

**Discussion by Wes-Mac** [ecn310/course-project-zipcentercrime#15] [General]
# For Each Member: #
- Something someone did to help the team this week:
- Something I could have done better for the team this week:
- An area / suggestion for team improvement:

**Comment by Wes-Mac** [comment]
### Weston 

- This week Leo helped our group by focusing on documentation and making sure that everything we needed was posted and logged about with links to what we did.
- something I could have done better this week was work more on my own documentation and remembering to post whenever work is done outside of class
- something we should work on as a group is begin figuring our everything that we must do with our data and have a plan in place so that we can get what we need done and have the correct documentation to go with it.

**Comment by Gersh305** [comment]
### Leo ###
- Something someone did to help the team this week: Rach dealt with the task of continuously reaching out to Professor Deza and waiting for our data to come back.
- Something I could have done better for the team this week: I could've done the group logs better before being reminded of it by Professor Buzard.
- An area / suggestion for team improvement: The team was thinking about changing our meeting time to Friday at 1:30 and making that change in the team contract. If it works for everyone I would recommend we change it to this.

**Comment by regaudre** [comment]
### Rachel 
- Leo has been organized with documentation and has been working on bettering our documentation process
- stay on top of tasks better and commit more time outside of class to working on different tasks
- I think we need to start assigning data tasks to each person in the group so we can start to make progress on some analysis

**Comment by sortizh** [comment]
## Sophia 
- Leo made the group log more specific with links, easy to navigate
- I could have learned how to geocode in my free time so I can progress with the data manipulation 
- Identify exact steps for data manipulation, document data manipulation better, and set deadlines for it/isolating separate tasks better

**Comment by sortizh** [comment]
Data manipulation next steps: 
Identify standardize radius levels. We will use those circles (circumference levels) around every drug treatment center and compare the number of 911 calls within one closer/smaller circle vs larger area circle (with the drug treatment center being the center of each circle during a singular year. 
Issues: Overlapping 911 ones as distance circles get larger, overlapping drug treatment centers. 
Wes will create a project on ArcGIS with only 2 layers for the same year and create a do file on how to do that. The year chosen will be before 2020
Outlier stuff - won't need to get rid of bc the data won't come up in the radius data
Put data sets by years into arc gis, get the radius levels and amount of 911 calls per radius level.
